leasing/management/commands/load_csv_leasearea.py

```diff
@@ -0,0 +1,211 @@
+import csv
+from collections import OrderedDict
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from leasing.enums import LeaseAreaType, LocationType
+from leasing.models.land_area import LeaseArea, LeaseAreaAddress
+from leasing.models.lease import Lease
+
+
+class Command(BaseCommand):
+    # Usage: `python manage.py load_csv_leasearea --csv path/to/file.csv`
+    help = "Load data from a CSV file to LeaseArea model and LeaseAreaAddress model"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--csv", type=str, required=False, help="The path to the CSV file"
+        )
```

Comment:
Ticket comment requests a new argument for Lease identifier.

Reply:
This was discussed in a meeting. I remember that the argument is not needed, because having the information in the CSV would be more transparent and easier to check for correctness.